src/gui/touchscreengui.h

```diff
@@ -246,6 +246,8 @@ class TouchScreenGUI
 	size_t m_move_id;
 	bool m_move_has_really_moved = false;
 	u64 m_move_downtime = 0;
+	// m_move_pos stays valid even after m_move_id has been released.
+	v2s32 m_move_pos;
```

Comment:
C++ question: Is this be initialized to a zero vector because of vector2d's default constructor? Or is this uninitialized?

Reply:
There's default initialization and value initialization. In this case, `m_move_pos` is default initialized. (`v2s32 m_move_pos{};` would value initialize.)
Value initialization first fills the thing with zeros, and then default initializes. Default initialization just calls the default constructor.
As the default constructor sets it to 0, it's initialized to 0.

Reply:
Thank you 👍